routes/index.js

```diff
@@ -95,32 +93,9 @@ module.exports = function(utils, nunjucksEnv, appName) {
         res.render('friendlycode/templates/publish-dialog.html');
       });
 
-      app.get( '/help-msg', function( req, res ) {
-        res.render('friendlycode/templates/help-msg.html');
-      });
-
-      app.get( '/error-msg', function( req, res ) {
-        res.render('friendlycode/templates/error-msg.html');
-      });
-
       app.get( '/nav-options', function( req, res ) {
```

Comment:
This is the only file we still use from the `friendlycode/templates` folder, and it doesn't need it's own middleware handler. If you stuff the file in `/views` and change the reference where it's rendered in `bramble.html` that should be enough to make the entire `friendlycodeRoutes` function unnecessary.